FactorioWebInterface/Program.cs

```diff
@@ -95,7 +97,68 @@ private static void SeedData(IHost host)
 
                 roleManager.CreateAsync(new IdentityRole(Constants.RootRole));
                 roleManager.CreateAsync(new IdentityRole(Constants.AdminRole));
+
+                _ = DefaultUserAsync(services);
+            }
+        }
+
+        private static async Task DefaultUserAsync(IServiceProvider services)
+        {
+            DefaultAdminAccountOption option = services.GetRequiredService<IOptions<DefaultAdminAccountOption>>().Value;
+
+            Console.WriteLine(option.Enabled.ToString());
+
+            String id = DefaultAdminAccountOption.DefaultAdminAccount;
+            var _userManager = services.GetRequiredService<UserManager<ApplicationUser>>();
+            ApplicationUser userResult = await _userManager.FindByIdAsync(id);
+
+            if (userResult != null)
+            {
+                var deleteResult = await _userManager.DeleteAsync(userResult);
+                if (!deleteResult.Succeeded)
+                {
+                    var lockoutResult = await _userManager.SetLockoutEnabledAsync(userResult, true);
```

Comment:
Hmm, I'm not sure if locking is worth it. Although I don't know under what conditions deleting would fail.

Reply:
Me neither, just wanted to be extra precautious, and I couldn't figure out how to set the user as suspended.

Reply:
It's probably not worth worrying about locking/suspending, if the delete fails the locking probably would as well.

For what it's worth I think this is how you suspend, but I haven't tried it:
```csharp
user.Suspended = true;
var result  = await _userManager.UpdateAsync(user);
```

Reply:
Yeah that is the same way I came up with after taking a deeper look at Identity on ASP.NET core